### DISCUSSION TITLE
Trouble loading recordings

### DISCUSSION THREAD

**Discussion by alexxnz** [nirgoren/Eddienput#2] [Technical support]
This is kind of an odd issue.

I noticed that I have issues loading recordings, but only after I launch the application using Wox ([https://github.com/Wox-launcher/Wox](https://github.com/Wox-launcher/Wox)) which is basically an application launcher for Windows.
If I launch Eddienput from Windows Explorer, I can load recordings without any issues.

I know Wox is launching Eddienput from the same location I open it in Windows Explorer since it shows the path underneath, so I can't figure out what would prevent the application from loading the config files (which is basically the error that I get).

Any ideas?

**Comment by nirgoren** [comment]
It's possible that Wox somehow messes with the current working directory.
You can use the absolute path to the config file in the recording file and it should work (so something like 
` C:\...\Eddienput\configs\gg.json` )
